subprojects/diagnostics/src/main/java/org/gradle/api/reporting/dependencies/internal/JsonProjectDependencyRenderer.java

```diff
@@ -140,30 +138,13 @@ private void renderProject(Project project, JsonBuilder json) {
         Map<String, Object> projectOut = Maps.newLinkedHashMap();
         projectOut.put("name", project.getName());
         projectOut.put("description", project.getDescription());
-        projectOut.put("configurations", createConfigurations(project));
+        projectOut.put("configurations", createConfigurations(configurations));
         overall.put("project", projectOut);
 
         json.call(overall);
     }
 
-    private List<Configuration> getConfigurationsWhichCouldHaveDependencyInfo(Project project) {
```

Comment:
Done in the caller now: 
https://github.com/gradle/gradle/pull/23366/files#diff-0c32dae86deab611a7c6fff2825d5ebe8b935488a33251962dd11ea0a46858adR189